### ISSUE TITLE
Not working with latest angular/ui-bootstrap (fix included)

### ISSUE THREAD

**Issue by defkev** [popdevelop/angular-splash-demo#3]
Love the minimalism of the splash as a landing page.

But the code needs some love too work with Angular UI Bootstrap 0.13+.

Since Google has moved all animation code out of the core into a seperate module you will need too source angular-animate.js in index.html below angular:

``` html
...
<script type="text/javascript" src="../angular-animate.min.js"></script>
...
```

and load the module in splash.js like so:

``` js
...
angular.module('ui.splash', ['ui.bootstrap','ngAnimate'])
...
```

Furthermore the splash/index.html windowTemplate will need some additions too get the splash too show and, more importantly, animate:

``` js
...
'<section modal-render="{{$isRendered}}" class="splash" modal-in-class="splash-open" ng-style="{\'z-index\': 1000, display: \'block\'}" ng-click="close($event)">' +
...
```

Cheers

**Comment by wallin** [comment]
Hi, thanks for reporting and suggesting a fix. 
However, I couldn't get it to work by using your fix. Do you have a working code example in jsbin or something?

**Comment by defkev** [comment]
Sure.

Here's my splash.js taken from production (just ignore the splash.result.then() function, i use it to store the users 'acknowledgement' of the splash, so he doesn't get 'splashed' again on page reload)

``` javascript
// Module for the demo
angular.module('splashDemo', ['ui.splash'])
.controller('MainCtrl', ['$splash', '$http', function ($splash, $http) {
  this.openSplash = function () {
    var container = angular.element(document.querySelector('.container'));
    container.addClass('blur')
    var splash = $splash.open();
    splash.result.then(function () {
        $http({
            url: url,
            method: 'POST',
            data: 'splashed=' + true,
            headers: {'Content-Type' : 'application/x-www-form-urlencoded'}
        })
    }).finally(function () {
        container.removeClass('blur')
    })
  };
}]);

// Re-usable $splash module
angular.module('ui.splash', ['ui.bootstrap','ngAnimate'])
.service('$splash', [
  '$modal',
  '$rootScope',
  function($modal, $rootScope) {
    return {
      open: function (attrs, opts) {
        var scope = $rootScope.$new();
        angular.extend(scope, attrs);
        opts = angular.extend(opts || {}, {
          backdrop: false,
          scope: scope,
          templateUrl: 'inc/splash/content.php',
          windowTemplateUrl: 'splash/index.html'
        });
        return $modal.open(opts);
      }
    };
  }
])
.run([
  '$templateCache',
  function ($templateCache) {
    $templateCache.put('splash/index.html',
      '<section modal-render="{{$isRendered}}" class="splash" modal-in-class="splash-open" ng-style="{\'z-index\': 1000, display: \'block\'}" ng-click="close($event)">' +
      '<div class="splash-bg"></div>' +
      '  <div class="splash-inner" ng-transclude></div>' +
      '</section>'
    );
  }
]);
```

**Comment by wallin** [comment]
Thanks! Did you do any modifications to the `templateUrl`, ie. `inc/splash/content.php` in your case?

**Comment by defkev** [comment]
Yup, its only a PHP/HTML snippet, this i source it externally.
The vars get populated based on the users geographical location, i left out this stuff, the rest is basically just the container:

``` php
...
<div class="splash-content">
<h1><?php echo $header;?>!</h1>
<p><?php echo $message;?>.</p>
<button class="btn btn-lg btn-outline" ng-click="$close()"><?php echo $button;?></button>
</div>
```

**Comment by wallin** [comment]
Fixed with 30e978a8c3d00a1381d042b231f15e77575ccec9

**Comment by cbaziotis** [comment]
+1
